impl/client/src/main/javascript/web/pentaho/ccc/visual/Treemap.js

```diff
@@ -16,12 +16,35 @@
 define([
   "pentaho/module!_",
   "./Abstract"
-], function(module, BaseView) {
+], function (module, BaseView) {
 
   "use strict";
 
   return BaseView.extend(module.id, {
-    _cccClass: "TreemapChart"
-  })
-  .implement(module.config);
+    _cccClass: "TreemapChart",
+
+    _roleToCccRole: {
+      "rows": "category",
+      "multi": "multiChart",
+      "size": "size"
+    },
+
+    _genericMeasureCccVisualRole: "size",
```

Comment:
Didn't notice before, but this configuration should have no effect here. The pair property, `_genericMeasureDiscrimCccVisualRole`, is not being configured, plus, the above mentioned `size` CCC visual role, is mapped to the `size` Viz visual role which is configured to accept a single field. We can talk more if you wish to enable this feature.